src/Sylius/Bundle/AdminBundle/templates/product/generate_variants/header/breadcrumbs.html.twig

```diff
@@ -1,8 +1,8 @@
-{% import '@SyliusAdmin/shared/helper/breadcrumbs.html.twig' as _breadcrumbs %}
+{% import '@SyliusAdmin/shared/helper/breadcrumbs.html.twig' as breadcrumbs %}
 
 {% set product = hookable_metadata.context.resource %}
 
-{{ _breadcrumbs.default([
+{{ breadcrumbs.default([
```

Comment:
Similar case as w/ accordion; the same thing is also used in `translations`, `channel_pricings` and probably more places